apps/files_versions/lib/Events/VersionRestoredEvent.php

```diff
@@ -0,0 +1,35 @@
+<?php
+/**
+ * SPDX-FileCopyrightText: 2017 Nextcloud GmbH and Nextcloud contributors
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ */
+namespace OCA\Files_Versions\Events;
+
+use OCP\EventDispatcher\Event;
+use OCA\Files_Versions\Versions\IVersion;
+
+/**
+ * Class VersionRestoredEvent
+ *
+ * Event that is called after a successful restore of a previous version
+ *
+ * @package OCA\Files_Versions
+ */
+class VersionRestoredEvent extends Event {
+	/**
+	 * VersionRestoredEvent constructor.
+	 *
+	 * @param IVersion $version
+	 */
+	public function __construct(private IVersion $version) {
+	}
+
+	/**
+	 * get Node of the file which should be versioned
+	 *
+	 * @return IVersion
+	 */
```

Comment:
You do not need to state param or return types if there is no additional information when using strong typing for the method.
Also the doc seems to be off.

```suggestion
	/**
	 * Version that was restored
	 */
```